CODEOWNERS

```diff
@@ -76,6 +76,7 @@ Kconfig*                                  @tejlmand
 /samples/bluetooth/                       @joerchan @lemrey @carlescufi
 /samples/bluetooth/mesh/                  @trond-snekvik @joerchan
 /samples/bootloader/                      @hakonfam @ioannisg
+/samples/crypto/aes/					  @peknis
```

Comment:
FYI, this is done only for CI issues, obviously a different code owner should be put here